main.py

```diff
@@ -123,6 +123,7 @@ def resize(viewer):
 argparser.add_argument("-v", "--visualize", help="Visualize the kernels we are going to draw", action="store_true")
 argparser.add_argument("-b", "--bidir", help="Use bidirectional path tracing instead of path tracing for "
                                                  "incompleteness images", action="store_true")
+argparser.add_argument("-i", "--inc", help="output incompleteness scenes", action="store_true")
```

Comment:
This mode is for only outputting the incompleteness images?

Reply:
Yes, since we don't need it all the time.